### PR TITLE
Remove preseq from blacklist, bump to 2.0.3

### PR DESCRIPTION
**Pull request by dpryan79**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
@bgruening This removes one of the things you put on the blacklist a while back. I don't recall the exact reasons it was put on, but can you have a look to see if there's something related to that that I missed?

File: recipes/preseq/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-  version: 2.0.2
+  version: 2.0.3
 
 source:
-  url: https://github.com/smithlabcode/preseq/releases/download/v2.0.2/preseq_v2.0.2.tar.bz2
-  fn: preseq_v2.0.2.tar.bz2
-  md5: 9f2a7b597c9f08b821db6ee55e2ea39c
-  patches:
-    - makefile.patch
+  git_url: https://github.com/smithlabcode/preseq
```
Comment (bgruening) [inline recipes/preseq/meta.yaml]: Is there any reason you can not use the release tarball? https://github.com/smithlabcode/preseq/releases/download/v2.0.3/preseq_v2.0.3.tar.bz2
It seems the submodules are included.

Comment (dpryan79) [reply recipes/preseq/meta.yaml]: When I tried that it didn't seem to work, but I'll give it a whirl again.

Comment (dpryan79) [reply recipes/preseq/meta.yaml]: I must have used the master branch tarball before.

**Pull request merged**